packages/wonder-blocks-dropdown/src/components/__tests__/action-item.test.js

```diff
@@ -28,20 +26,6 @@ describe("ActionItem", () => {
         expect(document.querySelectorAll("a")).toHaveLength(1);
     });
 
-    it("should render a Link if there's a router", () => {
-        // Arrange
-
-        // Act
-        const wrapper = mount(
-            <MemoryRouter>
-                <ActionItem href="/foo" label="Example" />
-            </MemoryRouter>,
-        );
-
-        // Assert
-        expect(wrapper.find(Link)).toHaveLength(1);
-    });
```

Comment:
**suggestion (non-blocking):** This test could be ported by mocking `react-router-dom`.

Reply:
Do you have an example of how that test would look like? you mean something like doing some client-side navigation to verify that the `Link` works as expected? or something completely different?

Reply:
See https://github.com/Khan/webapp/pull/11326/files#diff-a38bc63689752bc546ffae6d8a74d17f7c502b0aeacdd9dec7816baf8ffb1140.  You might not need to mock the whole module in order to mock Link.